logging/include/RialtoLogging.h

```diff
@@ -55,6 +57,7 @@ extern "C"
         RIALTO_COMPONENT_IPC,
         RIALTO_COMPONENT_SERVER_MANAGER,
         RIALTO_COMPONENT_COMMON,
+        RIALTO_COMPONENT_EXTERNAL, // External component, like e.g. GStreamer. Keep it as last component.
```

Comment:
It's also my suggestion only, it doesn't have to be last. I added this comment here, because EXTERNAL is not RIALTO component. I can also remove this comment if you don't like it.